src/library/features/crates/cratetreemodel.cpp

```diff
@@ -31,28 +31,26 @@ void CrateTreeModel::fillTree(const QStringList& idPaths) {
         if (child.getName() == "") {
             continue;
         }
-        m_treeCrates.insert(child.getId(), // key is the id of child
+        treeCrates.insert(child.getId(), // key is the id of child
                             // value is the treeItem returned by asdasdasppendChild()
-                            m_treeCrates.value(parent.getId())->appendChild(child.getName(),
+                            treeCrates.value(parent.getId())->appendChild(child.getName(),
                                                                     child.getId().toInt()));
 
-        //        m_treeCrates.value(child.getId())->setLabel(idPath);
+        //        treeCrates.value(child.getId())->setLabel(idPath);
     }
 }
 
 void CrateTreeModel::reloadTree() {
     const QStringList idPaths = m_pTrackCollection->crates().collectIdPaths();
-
-    // empty crate map
-    m_treeCrates.clear();
+    QMap<CrateId,TreeItem*> treeCrates;
```

Comment:
This is not needed, is it? Don't you just need a local variable in the fillTree method?